package.json

```diff
@@ -7,7 +7,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "test": "NODE_ENV=test yarn undo:migration && NODE_ENV=test yarn migrate && NODE_ENV=test yarn undo:seed && NODE_ENV=test yarn seed && NODE_ENV=test nyc --reporter=html --reporter=text mocha --exit --require @babel/register --require @babel/polyfill src/test/index.js",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
-    "dev-start": "NODE_ENV=development nodemon --exec babel-node src/index.js",
+    "dev-start": "nodemon --exec babel-node src/index.js",
```

Comment:
Great job here, @victor-shagor. Was there a reason why the script to start the server was changed?

Reply:
yes @ChiamakaObitube the script doesn't work on my local system, but i will change it back

Reply:
done.. thanks